server/publications/collections/orders-publications.app-test.js

```diff
@@ -65,8 +64,7 @@ describe("Order Publication", function () {
       const collector = new PublicationCollector({ userId: Random.id() });
       collector.collect("Orders", (collections) => {
         expect(collections.Orders.length).to.equal(0);
-        done();
-      });
+      }).then(() => done(/* empty */), done);
```

Comment:
putting `done()` after an `expect` statement yields a timeout error when the expectation failed. this isn't super useful, so use the Promise provided by `PublicationCollector#collect` to show better error messages.